portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -53,8 +64,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       String email = (String) entity.getProperty("email");
       long timestampMillis = (long) entity.getProperty("timestampMillis");
       String commentInput = (String) entity.getProperty("commentInput");
+      String fileUrl = (String) entity.getProperty("fileUrl");
 
-      Comment comment = new Comment(id, name, email, timestampMillis, commentInput);
+      Comment comment = new Comment(id, name, email, timestampMillis, commentInput, fileUrl);
       comments.add(comment);
```

Comment:
Although the variables are only used once, I feel like inlining them would be less readable / more complicated because there are so many...

Reply:
fine either way for me, but if you're inlining you could format it one parameter per line

```
Comment comment = new Comment(
   entity.getKey().getId(),
   entity.getProperty("name"),
   entity.getProperty("email"),
  ...
   entity.getProperty("fileUrl")
);
```

In this case because the entity.getProperty parameter is basically the same as the variable, it doesn't make it too unreadable since we can still understand contextually what the propertiees/parametes are.

Reply:
Ah I see, I can make that change